loops_lab.js

```diff
@@ -0,0 +1,111 @@
+//1. Write a `while` loop and a `for` loop that takes the variable
+//  "num" and logs all the numbers, descending, between "num" and 1.
+console.log("-----while loop-------")
+let num = 15
+while (num>=1) {
+  console.log(num)
+num--
+}
+console.log("-----for loop-------")
+for (let newNum= 15; newNum >= 1; newNum--) {
+  console.log(newNum)
+}
+
+//2. Write a `while` loop and a `for` loop that takes the variable "max",
+//and iterates over all numbers from 0 to "max". For each iteration, it will
+//check if the current number is even or odd, and log that to the screen
+//(e.g. "2 is even")
+
+console.log("-----while loop-------")
+let max = 0;
```

Comment:
Would max be maximum number when it is assigned as 0 in this case? Just some minor variable naming issue